dev/tools/gen_defaults/lib/slider_template.dart

```diff
@@ -39,16 +39,17 @@ class _${blockName}DefaultsM3 extends SliderThemeData {
   Color? get disabledSecondaryActiveTrackColor => _colors.onSurface.withOpacity(0.12);
 
   @override
-  Color? get activeTickMarkColor => ${componentColor('$tokenGroup.with-tick-marks.active.container')};
+  Color? get activeTickMarkColor => _colors.${getToken("$tokenGroup.disabled.stop-indicator.color-selected")};
 
   @override
-  Color? get inactiveTickMarkColor => ${componentColor('$tokenGroup.with-tick-marks.inactive.container')};
+  // TODO(tahatesser): Update this hard-coded value to use the correct token value.
+  Color? get inactiveTickMarkColor => _colors.primary;
```

Comment:
I think this token is available

Reply:
I checked, it seems incorrect it uses secondary container color which makes inactive tick marks invisible on the inactive track.  I will file an issue as this lands.